modules/rules_swift/1.2.0/MODULE.bazel

```diff
@@ -0,0 +1,31 @@
+module(
+    name = "rules_swift",
+    compatibility_level = 1,
+    repo_name = "build_bazel_rules_swift",
+    version = "1.2.0",
+)
+
+bazel_dep(name = "bazel_skylib", version = "1.1.1")
+bazel_dep(name = "apple_support", repo_name = "build_bazel_apple_support", version = "1.3.1")
+bazel_dep(name = "rules_cc", version = "0.0.2")
+bazel_dep(name = "protobuf", repo_name = "com_google_protobuf", version = "3.19.2")  # To be removed once rules_proto is bzlmod-ready.
```

Comment:
rules_proto is available in the BCR and could be added as a module dependency.

Reply:
The only version available is 4.0.0 which doesn't depend on protobuf itself, so this is why this is needed: https://github.com/bazelbuild/bazel-central-registry/blob/main/modules/rules_proto/4.0.0/MODULE.bazel

If we check-in a newer version of 4.0.0, we could probably specify it as a module dependency.
https://github.com/bazelbuild/rules_proto/blob/master/MODULE.bazel

Reply:
You can certainly keep it this way for the first version, but having an updated version of rules_proto would of course prevent potentially duplicating it in the dependency tree.